lib/saito/ui/game-playerbox/game-playerbox.js

```diff
@@ -51,14 +87,32 @@ class GamePlayerBox {
     }
 
 
+    /*
+    Hide all playerboxes (selected by ID)
+    */
     hide() {
       try {
         for (let i = 1; i <= this.game_mod.game.players.length; i++) {
-	  let player = this._playerBox(i);
+	        let player = this._playerBox(i);
           if (document.getElementById(("player-box-"+player))) {
             document.getElementById(("player-box-"+player)).style.display = "none";
-	  }
-	}
+	        }
```

Comment:
If you throw this into [prettier](https://prettier.io/) the format could be more standardized.